app/src/main/java/com/example/myfridgehome/service/RecipeService.kt

```diff
@@ -9,8 +9,10 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class RecipeService {
+
+    //Creates recipes for application
     fun collectRecipes(): MutableLiveData<ArrayList<Recipe>> {
-        var _recipes = MutableLiveData<ArrayList<Recipe>>()
+        var recipes = MutableLiveData<ArrayList<Recipe>>()
```

Comment:
Renamed variable to follow naming convention used.

Reply:
I'd switch this around a bit.  _recipes is the naming convention for private variables, recipes is the convention for public or internal variables.  So... maybe make _recipes private, and then expose it through a public variable recipes.